client/src/game/unit.ts

```diff
@@ -169,6 +202,7 @@ export class Unit {
             ? src.weight
             : unit.inventory.items.toArray().map(x => x.weight).reduce((w, v) => w + v)
 
+        unit.region = region;
```

Comment:
Unit will be added into the region in another place (in the World class, when unit is placed into the region). This is not needed here.

Reply:
I did this change to not create units that don't have a region set. I had an issue that by creating a unit without a region, their move orders get processed and creation of path fails, as there is no starting point.

For a unit to exist outside of any region kinda doesn't make much sense anyway. If order processing will be different, then this change might not be needed, but for any order processing unit's region is essential.

Reply:
There can be different cases why we need a Unit, and the Single Responsibility Principle still applies :)

Reply:
I don't see how unit without a region has a usecase, but anyway this can be changed together with `updatePath` moved to a proper place outside of unit, when we have a order processing layer setup.